.github/workflows/publish.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   build:
     strategy:
       matrix:
-        game_version: ["1.20.1", "1.20.2", "1.20.4"]
```

Comment:
Why? The entire point of the stonecutter system is so it supports ALL the currently used game versions.